data/egg/everest/model/egg.yml

```
@@ -63,46 +63,37 @@ optimization:
   max_function_evaluations: 5000
   perturbation_num: 2 
   max_iterations:  10
-  max_batch_num:  10 
+  max_batch_num: 1 #10 
   speculative: True
 
-
 simulator:
   queue_system:
     name: lsf
     max_running: 200
 
 model:
-  realizations: [6,10,31,22,24,45,50,68,36,62]
-  data_file: r{{opm_initial_data}}
-
-
+  realizations: [6, 10, 31, 22, 24, 45, 50, 68, 36, 62]
```

```suggestion
  realizations: [6, 10, 31, 22, 24, 45, 50, 68, 36, 62]
  data_file: r{{flow_folder}}/model/EGG_MODEL_FLOW.DATA
```
I would keep it otherwise simulation data is not loaded to database.